gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -66,13 +66,13 @@ to generate set of instructions that includes a command block with the registrat
 == Register and Run with a Token as a Linux Service
 // end::token-linux-heading[]
 // tag::app-k8s-heading[]
```

Comment:
These headings are for general sections used for both connected and local modes.